src/main/java/eu/execom/pomodoroTeam/controllers/TeamController.java

```diff
@@ -116,11 +122,22 @@ public TeamController(TeamRepository teamRepository, UserRepository userReposito
         teamRepository.save(team);
         return new ResponseEntity<>(team, HttpStatus.OK);
     }
-}
-
-
-
-
-
-
 
+    /**
+     * remove user from team
+     *
+     * @param id
+     * @param user
+     * @return
+     */
+    @RequestMapping(method = RequestMethod.PUT, value = "/leaveTeam/{id}/user")
+    public ResponseEntity<TeamEntity> leaveUserFromTeam(@PathVariable Long id, @RequestParam Long user) {
+        TeamEntity team = teamRepository.getOne(id);
+        UserEntity us = userRepository.getOne(user);
+        List<UserEntity> users = team.getUser();
+        users.remove(us);
+        team.setUser(users);
```

Comment:
No need to call this again, since you are working with references (you have a reference to a list of users, once you remove a user from the list, it will be updated automatically on the team).
Also, it would be more appropriate to introduce a service and put the removing logic there.